client/src/pages/live-Invocations/ImpotTestStepDialog.tsx

```diff
@@ -1,13 +1,14 @@
-import { useState } from "react";
+import React, { useState } from "react";
 import GenericEditingModal from "../../components/project-selection/GenericEditingModal";
 import { useSnackbar } from "../../hooks/useSnackbar";
 import { useFlowStore } from "../../stores/flowStore";
 import { FlowSelector } from "../flows/TestsTab";
 import { InvocationType } from "../sniffers/types";
+
 interface ImportTestStepDialogProps {
   open: boolean;
   handleClose: () => void;
-  invocation?: InvocationType;
+  invocation?: InvocationType | InvocationType[];
```

Comment:
I would change it to be invocations and accept only a list of invocations.
Invocation is singular but you accept a list as well. This is very confusing.

Reply:
Changed it